old-ui/app/components/app-bar/networks-menu.js

```diff
@@ -110,7 +112,12 @@ class NetworksMenu extends Component {
         <DropdownMenuItem
           key={networkObj.providerName}
           closeMenu={() => props.updateNetworksMenuOpenState(!isOpen)}
-          onClick={() => props.setProviderType(networkObj.providerName)}
+          onClick={() => {
+            if (networkObj.providerName === 'rsk' || networkObj.providerName === 'rsk_testnet') {
```

Comment:
Will this providerName always be lowercase? Maybe we can put a .toLowerCase just to be sure

Reply:
It's defined by us in a nifty enumerated so yeah it's fixed and it doesn't change